test_rigs/parsing-tests/unit-tests.ref

```diff
@@ -25598,7 +25662,7 @@
             "        BSD_CLIENT:'bsd-client'  <== mode:DEFAULT_MODE",
             "        (null_rest_of_line",
             "          SERVER:'server'  <== mode:DEFAULT_MODE",
-            "          VARIABLE:'url'  <== mode:DEFAULT_MODE",
+            "          URL:'url'  <== mode:DEFAULT_MODE",
```

Comment:
I am not sure why this got changed to URL, I didn't change this rule. I am guessing this happened because I added `URL` token in `CiscoLexer` but don't know why